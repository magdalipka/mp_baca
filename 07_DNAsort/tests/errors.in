11
AT
ATG
CCC
DDATGCAAATATGAFF
ATGCAACCCTATTAA
ATGTGATAA
ATGTACTAAATG
ATGTTAATGA
ATGTTTAATGA
CCCATGCAATAGATT
TGATGGCACATGCCCTAA

