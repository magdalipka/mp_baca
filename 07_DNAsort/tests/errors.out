Wrong DNA sequence.
Wrong DNA sequence.
Wrong DNA sequence.
Wrong character in DNA sequence.
CAACCCTAT
More than one START/STOP codon.
TAC
Wrong DNA sequence.
Wrong DNA sequence.
CAA
Wrong DNA sequence.
